pkg/cmd/roachtest/tests/failure_injection.go

```diff
@@ -617,6 +661,31 @@ func setupFailureSmokeTests(
 	if err := c.Install(ctx, t.L(), c.CRDBNodes(), "vmtouch"); err != nil {
 		return err
 	}
+
+	// Setup cgroup_exporter as a scrape target in prometheus.
+	// TODO(darryl): Once #143404 is complete, we should switch all cgroup
+	// performance assertions to use prometheus metrics instead of the current
+	// approach of sampling cgroup directly. There's nothing blocking us from
+	// using clusterstats to query for metrics, but the solution proposed in #143404
```

Comment:
I do have a working example of this test using the clusterstats package the query for read/write bytes if we really want to go that route. But IMO there's such much boilerplate required by clusterstats that I don't think it's necessarily a cleaner solution than what we have right now.